src/Bench/JmsMinimalBenchmark.php

```diff
@@ -40,4 +40,16 @@ public function serialize(Forum $data): void
     {
         $this->serializer->serialize($data, 'json');
     }
+
+    public function getPackageName(): string
+    {
+        return 'jms/serializer';
+    }
+
+    public function getNote(): string
+    {
+        return <<<'NOTE'
+Serialize object graphs with default handlers disabled
```

Comment:
Can you please put here: "Serialize object graphs with a minimal set of handlers"